docs/guides/portals/sickle-cell.md

```diff
@@ -1,4 +1,4 @@
-The Sickle Cell Genomics Portal contains two viewers for the exploration of data from the  [Sickle Cell Genome Project (SGP)](../glossary/data-access-unit#sickle-cell-genome-project-sgp) dataset.
```

Comment:
This link was more specific and seemed to work, what was the thought behind changing to just land them on all datasets?

Reply:
No that link was broken. I had to change the link because glossary.md is a page by itself now with sections for data access agreement, data access committee, data access units (with data units just bolded), and embargo date.

Before glossary was a header with individual pages for data access agreement, data access committee, data access units (with data units as sections), and embargo date. We decided to switch to the current version so as not to clutter up the side bar.